cmd/cmd.go

```diff
@@ -99,6 +99,7 @@ func initConfig() {
 			fmt.Fprintln(os.Stderr, err)
 		}
 	}
+	fmt.Println("Using config file:", viper.ConfigFileUsed())
 }
```

Comment:
Error generating comment for file cmd/cmd.go: Post "https://api.openai.com/v1/chat/completions": proxyconnect tcp: dial tcp 127.0.0.1:10838: connectex: No connection could be made because the target machine actively refused it.